src/Spies/GlobalSpies.php

```diff
@@ -105,6 +109,12 @@ public static function call_original_global_function( $function_name, $args ) {
 		if ( ! isset( self::$redefined_functions[ $function_name ] ) ) {
 			return;
 		}
+
+		// skip if the function did not exist before (i.e. had no "original" function)
+		if ( isset( self::$generated_functions[ $function_name ] )) {
```

Comment:
Very minor spacing here (if only we had prettier):
```suggestion
		if ( isset( self::$generated_functions[ $function_name ] ) ) {
```